pkgs/development/python-modules/psycopg/default.nix

```diff
@@ -34,13 +34,13 @@
 
 let
   pname = "psycopg";
-  version = "3.2.3";
+  version = "3.2.4";
 
   src = fetchFromGitHub {
     owner = "psycopg";
-    repo = pname;
-    tag = version;
-    hash = "sha256-vcUZvQeD5MnEM02phk73I9dpf0Eug95V7Rspi0s6S2M=";
+    repo = "psycopg";
+    tag = "pool-${version}";
```

Comment:
What's the reason of change to the `pool-<version>` tag ? It should be at mentioned in the commit description. Or even better, moved to the separate commit.